assets/stylesheets/shared/_colors.scss

```diff
@@ -76,3 +76,12 @@ $color-skype: #00AFF0;
 $color-telegram: #0088cc;
 $color-whatsapp: #43d854;
 $color-wordpressOrg: #585c60;
+
+// New Signup Colors
```

Comment:
If we might use these colors elsewhere in the future then "signup" might not be the best prefix. Is there a name for this rebranding we could use here?